account_invoice_view_payment/README.rst

```diff
@@ -0,0 +1,73 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+============================
+Account Invoice View Payment
+============================
+
+This module allows users to access directly to the payment from an invoice
+when registering a payment or afterwards.
+
+The option to open the payment when it's being registered is useful
+when the user needs to do a follow-up step on the payment, such as printing
+the associated check.
+
+Configuration
+=============
+
+Only users assigned to the 'Accounting & Finance' groups 'Accountant' or
```

Comment:
This has changed in V11 CE. Please change to Users assigned to the group "Billing" can display the payments from the invoice.

Reply:
Updated.
Many Thanks @jbeficent  for the review.